### ISSUE TITLE
Twitter Card for Manual entries looks terrible

### ISSUE THREAD

**Issue by ry** [denoland/dotland#1803]
Try plugging a manual entry (like https://deno.land/manual@v1.12.1/tools/dependency_inspector) into the twitter card validator 
(https://cards-dev.twitter.com/validator)

<img width="1025" alt="Screen Shot 2021-07-21 at 1 16 47 PM" src="https://user-images.githubusercontent.com/80/126531439-c8be0299-75bb-4bb5-a062-f81eee9c758e.png">

**Comment by meszarosdezso** [comment]
I took a look into this but soon I realized that the content for the currently read manual page is fetched on the client side - and afaik twitter/og meta tags need to be server side rendered.

I guess we could get the current page's title and the first paragraph of the content (for `twitter:description`) in the `getServerSideProps` of the `RestPage` component and pass them to the `Manual` component but I don't want to start adding that without discussing it here, because maybe that'd be too much overhead, or there is a simpler solution for this. ( I guess converting the _whole manual_ to be built during build time with `getStaticPaths`+`getStaticProps` would increase the build time and page size with a lot? )

( cc @lucacasonato, maybe you have a better idea on how to deal with this(: )
